src/core/experimental/memutils.d

```diff
@@ -0,0 +1,251 @@
+/**
+ * Pure D replacement of the C Standard Library basic memory building blocks of string.h
+ * Source: $(DRUNTIMESRC core/experimental/memutils.d)
+ */
+module core.experimental.memutils;
+
+/*
+  If T is an array, set all `dst`'s bytes
+  (whose count is the length of the array times
+  the size of the array element) to `val`.
+  Otherwise, set T.sizeof bytes to `val` starting from the address of `dst`.
+*/
+void memset(T)(ref T dst, const ubyte val)
```

Comment:
Is this undocumented (ddoc) on purpose?

Reply:
You mean it should be:
`
/**
 * ...
 */`
?

Reply:
Yes, and a `Params` section (https://dlang.org/spec/ddoc.html#standard_sections) is probably best too.

Reply:
Hmm, thanks. I'm not very accustomed yet to the logistics of contributing to D.

Reply:
Probably you can add `scope` to the reference? Explicit `nogc` `nothrow` may be good too.

Reply:
Yes, I'm not very familiar with `scope` but `nogc`, `nothrow` is good.